multiversx_sdk/account_management/account_controller.py

```diff
@@ -18,6 +18,8 @@ def create_transaction_for_saving_key_value(
         sender: IAccount,
         nonce: int,
         key_value_pairs: dict[bytes, bytes],
+        gas_limit: Optional[int] = None,
+        gas_price: Optional[int] = None,
```

Comment:
Breaking change, but I think we can live with it (and publish as minor version).